test/src/code_field/tabs_to_spaces_test.dart

```diff
@@ -21,148 +21,114 @@ class Foo {
 const _spaceCount = 2;
 
 void main() {
+  void expectWillNotChanged(TextEditingValue value) {
```

Comment:
1. Please move this function inside the first group. Otherwise it is unclear what it actually does that should cause no change.
2. Please rename to `expectSame`